science/lenstool/Portfile

```diff
@@ -0,0 +1,39 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           compilers 1.0
+
+name                lenstool
+version             7.1.1
+categories          science
+license             GPL
+platforms           darwin
+maintainers         {lam.fr:eric.jullo @ejullo} openmaintainer
+
+description         A gravitational lensing software for modeling mass distribution of galaxies and clusters (strong and weak regime).
+
+long_description    LENSTOOL package contains a C library and several \
+                    executables for creating, and post-processing \
+                    gravitational lensing models. LENSTOOL lens model, \
+                    processing steps, and input data are listed in an ASCII \
+                    parameter file. The lenstool executable parses this file \
+                    and performs the successive requested processing steps. \
+                    Perl scripts provided in the package allow to visualize \
+                    output results in the graphical interface DS9 provided by \
+                    Smithsonian Astrophysical Observatory (SAO). In case of \
+                    MCMC model optimization, MCMC samples are delivered in \
+                    an output ASCII file. MCMC samples can be post-processed \
+                    by several additional executables provided in the package. 
+
+homepage            https://projets.lam.fr/projects/lenstool
+master_sites        https://projets.lam.fr/attachments/download/5851
+distname            ${name}-${version}
+worksrcdir          ${name}-${version}
+
+checksums           sha256 272aed5e5964b8b941c916be98f13a7f134f3a785681c5b6199ab0438114f5ac
+
+depends_lib         port:gsl \
+                    port:wcslib \
+                    port:cfitsio  
+
+configure.args      --with-cfitsio-prefix=${prefix} --with-wcslib-include-path=${prefix}/include/wcslib --with-wcslib-lib-path=${prefix}/lib --with-gsl-prefix=${prefix} 
```

Comment:
It is easier to read these if you list one arg per line like dependencies are listed.